library/Vanilla/EmbeddedContent/AbstractEmbedFactory.php

```diff
@@ -61,7 +61,8 @@ public function canHandleUrl(string $url): bool {
         if ($path === null) {
             $pathMatches = $this->canHandleEmptyPaths;
         } else {
-            $pathMatches = (bool) preg_match($this->getSupportedPathRegex($domain), $path);
+            $regex = $this->getSupportedPathRegex($domain);
+            $pathMatches = (bool) preg_match($regex, $path);
```

Comment:
I don't see why we need to change anything here.
But if we do - I would prefer to avoid casting types - it is tricky and suspicious.
```suggestion
            $pathMatches = preg_match($regex, $path) === 1;
```

Reply:
The change was actually to assist with debugging (so I could put a breakpoint and look at the generated regex).